etc/fetchers/fetch-docs.js

```diff
@@ -0,0 +1,16 @@
+const cheerio = require("cheerio");
+const fs = require("fs");
+const path = require("path");
+const fetch = require("node-fetch");
```

Comment:
In #19, @nibraswastaken [tried to replace](https://github.com/kawalcovid19/wargabantuwarga.com/pull/19#discussion_r665538380) `node-fetch` with `cross-fetch`.

I think it's a good idea due to [its universal claim](https://www.npmjs.com/package/cross-fetch), so we can use it later in the browser environment when we need it.
Could you please do the replacement as well in this PR? Thanks!

Reply:
Fixes at ffe2fd1